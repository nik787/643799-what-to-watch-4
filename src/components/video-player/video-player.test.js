import React from "react";
import renderer from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

const videoData = {
  poster: `https://api.adorable.io/avatars/128/1`,
  preview: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
};

describe(`VideoPlayer`, () => {
  it(`Should render correctly`, () => {
    const tree = renderer
      .create(<VideoPlayer
        isPlaying={false}
        source={videoData.preview}
        poster={videoData.poster}
        isMuted={false}
      />, {
        createNodeMock: () => {
          return {};
        }
      })
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
