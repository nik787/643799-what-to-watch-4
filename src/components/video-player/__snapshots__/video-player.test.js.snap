// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer Should render correctly 1`] = `
<video
  height={175}
  onMouseOut={[Function]}
  onMouseOver={[Function]}
  width={280}
>
  Soory, your browser doesn't support video.
</video>
`;
